src/store/store.js

```diff
@@ -62,14 +62,21 @@ const saveStateToLocalStorage = (state) => {
  * @param {Object} initialState Initial Redux state (from initial-state.js)
  * @return {Object} Redux store
  */
-export default function configureStore(initialState) {
+export default function configureStore(initialState, dataType) {
   const composeEnhancers =
     window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
   const store = createStore(
     reducer,
     initialState,
     composeEnhancers(applyMiddleware(thunk))
   );
+
+  // dispatch the calculateGraph action to ensure the graph nodes still gets rendered
+  // on initial load if data is loaded via data prop instead of fetching from Rest API
+  if (dataType !== 'json') {
```

Comment:
I was testing this out by logging `dataType` and it was always coming back as `undefined`.

Reply:
good catch - this is due to the lack of the prop in `configureStore` as you've pointed before ( still pinching myself for forgetting to commit that change 😆 ) but now that it's pushed in, this would not be coming back as `undefined`.